huggle/welcomeinfo.cpp

```diff
@@ -12,7 +12,12 @@
 #include "ui_welcomeinfo.h"
 #include "configuration.hpp"
 #include "localization.hpp"
+#include <QMainWindow>

```

Comment:
Do we really need all these includes? QDebug and so on, QLibraryInfo, why do you include them? doesn't it compile without them as well?